### PR TITLE
Fixed majors issues

### PR DESCRIPTION
**Pull request by mickaelandrieu**
This is a "technical debt" pull request.

I don't know how to test to be sure this update wont break anything /c @crouillon 

Also, I have updated travis configuration to test PHP 7.0 and remove hhvm-nightly /c @eric-chau

### CODE REVIEW

**Comment by eric-chau** [comment]
100% agree with you about switching ``hhvm-nightly`` into ``php 7.0`` @mickaelandrieu.

**Comment by ndufreche** [comment]
Why switching `hhvm-nightly` to `php7`. We can just add `php7` and keep `hhvm-nightly`

**Comment by mickaelandrieu** [comment]
``hhvm-nightly`` is no more accepted by travis (we can keep hhvm)

File: Controller/FrontController.php [hunk 1]
```diff
@@ -302,11 +303,14 @@ public function defaultAction($uri = null, $sendResponse = true)
...
-                header('Location: '.$redirect);
-                exit();
+                $response = new RedirectResponse($redirect, 301, [
+                    'Cache-Control' => 'no-store, no-cache, must-revalidate',
+                    'Expires' => 'Thu, 01 Jan 1970 00:00:00 GMT',
+                    ]
+                );
+
+                $this->send($response);
+                $this->application->stop();
```
Comment (crouillon) [inline Controller/FrontController.php]: :+1:

**Comment by crouillon** [comment]
:+1:

**Comment by mickaelandrieu** [comment]
@crouillon thank you for review, I stay focused on this in case we have some weirds issues after this merge ==> Silver Rank is here :)

**Pull request merged**